xbmc/cores/RetroPlayer/savestates/SavestateDatabase.cpp

```diff
@@ -204,7 +204,7 @@ std::unique_ptr<ISavestate> CSavestateDatabase::RenameSavestate(const std::strin
 
   newSavestate->Finalize();
 
-  std::string path = savestatePath;
+  const std::string& path = savestatePath;
   if (!AddSavestate(path, "", *newSavestate))
```

Comment:
We can get rid of the `path` variable entirely, right?
```suggestion
  if (!AddSavestate(savestatePath, "", *newSavestate))
```

Reply:
I believe we can.

Reply:
I've removed the variable. Usually I only apply the fixes offered by clang-tidy.

Reply:
It's good that @neo1973 caught this then!